README.md

```diff
@@ -1,2 +1,38 @@
-# Phantom-Backend-Elites
+
```

Comment:
Good job @nirifabien1234, however this is not needed here in the Readme. We will find what to include in the readme, in the meantime, you can remove this.

Reply:
Do I need to delete all the texts that I have added to the ReadMe?